docs/src/modules/develop/pages/gke.adoc

```diff
@@ -0,0 +1,53 @@
+= Run on GKE
+
+include::ROOT:partial$attributes.adoc[]
+include::ROOT:partial$include.adoc[]
+
+== Cluster creation
+Create a GKE cluster. We recommend at least 6 vCPUs (ie, a node pool of 3 `n1-standard-2` nodes). Also ensure that the current user is a cluster admin.
+
+Detailed instructions for creating the GKE cluster can be found in the Knative documentation, follow all the steps up to (but not including) installing Knative.
+
+== Cassandra Set up
+If using an event sourced entity, install Cassandra. This can be done from the Google Marketplace, by visiting the Cassandra Cluster, selecting configure, selecting your GCloud project, and then installing it in the Kubernetes cluster you just created.
```

Comment:
Add the link to Cassandra Cluster in marketplace, like in the README?

```suggestion
If using an event sourced entity, install Cassandra. This can be done from the Google Marketplace, by visiting the https://console.cloud.google.com/marketplace/details/google/cassandra[Cassandra Cluster], selecting configure, selecting your GCloud project, and then installing it in the Kubernetes cluster you just created.
```